perception/sandbox/eyehandcal/scripts/collect_data_and_cal.py

```diff
@@ -208,6 +220,8 @@ def extract_obs_data_std(data, camera_index):
             print('stage 2 mean pixel error', pixel_error)
             if pixel_error > args.pixel_tolerance:
```

Comment:
if float(pixel_error) > float(args.pixel_tolerance):

Reply:
arg.pixel_error is now float as we added `type=float` in https://github.com/facebookresearch/fairo/pull/1113/files#diff-9c0bbc45d239fbb8bb96cb7e742d0ee57f2395e4eb9948f1d833cb5039e24072R130

Reply:
arg.pixel_error is now float as we added `type=float` in https://github.com/facebookresearch/fairo/pull/1113/files#diff-9c0bbc45d239fbb8bb96cb7e742d0ee57f2395e4eb9948f1d833cb5039e24072R130